utils/near-stable-hasher/README.md

```diff
@@ -0,0 +1,25 @@
+# `near-stable-hasher`
+
+`near-stable-hasher` is a library that is essentially a wrapper around, now deprecated, `std::hash::SipHasher`.
+Its purpose is to provide a stable hash function, which doesn't change depending on `rust_version`, `architecture`, `platform`,
+`time`, etc.
+
+In addition, note that `SipHasher` is deprecated since `Rust` `1.13.0`.
+Eventually `SipHasher` will be removed from `Rust`.
+We need to ensure, nothing breaks during this transition period.
```

Comment:
it seems that currently it is only used by network code, right?

Reply:
@mm-near Also 
`name = "near-vm-runner"`